File: src/main/java/com/wowo/wowo/repository/GroupFundRepository.java
package com.wowo.wowo.repository;

import com.wowo.wowo.model.GroupFund;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface GroupFundRepository extends JpaRepository<GroupFund, Long> {
    // Bạn có thể thêm các phương thức tùy chỉnh nếu cần
    List<GroupFund> findAllById(Iterable<Long> ids);
}

File: src/main/kotlin/com/wowo/wowo/contexts/groupfund/domain/repository/GroupFundRepository.kt
package com.wowo.wowo.contexts.groupfund.domain.repository

import com.wowo.wowo.contexts.groupfund.domain.GroupFund
import com.wowo.wowo.contexts.groupfund.domain.GroupId

interface GroupFundRepository {
    fun findById(id: GroupId): GroupFund?

    fun existsMember(groupId: GroupId, memberId: String): Boolean
}
